orchagent/pfcwdorch.cpp

```diff
@@ -560,11 +560,8 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
                 sai_serialize_object_id(queueId));
     }
 
-    // Create egress ACL table group for each port of pfcwd's interest
-    sai_object_id_t groupId;
-    gPortsOrch->createBindAclTableGroup(port.m_port_id, groupId, ACL_STAGE_INGRESS);
-    gPortsOrch->createBindAclTableGroup(port.m_port_id, groupId, ACL_STAGE_EGRESS);
-
+    // We do NOT need to create ACL table group here. It will be
```

Comment:
When we create PFC watchdog on port, we create ingress and egress ACL tables for each queue and bind the port to each of these tables. So, the required group ID of the port was being created here. Thats because we neither deleted a port (after initial creation) not deleted the ACL group ID created for the port. Now we delete the port and also delete the ACL group of the port. So, we don't have to explicitly create it here, it will be created when the first ACL table is bound to port.

Please go throught pfcwdorch.cpp and pfcactionhandler.cpp files, it gives you better picture.